README.md

```diff
@@ -1,100 +1,151 @@
 
-Let's say you want to run the `metadata_usage` example - then do this:
+Download a copy of the `credentials.json` file stored on Synapse by running the following command:
 
 ```bash
-python[3] examples/metadata_usage.py
+$ synapse get syn21088684
 ```
 
-### Configure Synapse Credentials
+The `credentials.json` file is required when you are using [`OAuth2`](https://developers.google.com/identity/protocols/oauth2) to authenticate with the Google APIs.
+
+Steps involved in the `OAuth2` [authorization flow](https://github.com/Sage-Bionetworks/schematic/blob/develop/schematic/utils/google_api_utils.py#L18):
 
-Download a copy of the `credentials.json` file (or the file needed for authentication using service account, called `quickstart-1560359685924-198a7114b6b5.json`) stored on Synapse, using the synapse client command line utility. The credentials file is necessary for authentication to use Google services/APIs. To do so:
+- The app (i.e., _schematic_) looks for a `token.pickle` file first. It is a file that is created when you have authorized your account to access Google APIs for the first time. If the file already exists, then it get the credentials from it, meaning you do not have to allow access several times.
+- If the `token.pickle` file does not exist, the app will load the credentials for you from the `credentials.json` file and create a `token.pickle` in your project root directory when you access any of the schematic services (e.g., through the Command Line Interface) and the authorization flow completes for the first time. For this:
+  - A URL will be prompted to you when you access a _schematic_ CLI utility. Copy the URL from the console and open it in your browser.
+  - If you are not already logged into your Google account, you will be prompted to log in. If you are logged into multiple Google accounts, you will be asked to select one account to use for the authorization.
+  - You might see a "Google hasn’t verified this app" warning from Chrome on your browser. To resolve this, click on "Advanced" and then "Quickstart (unsafe)". Select a Google account and authorize the _Quickstart_ script to access Google Drive/Sheets under your account.
 
 
-_Note: Make sure you have `download` access/permissions to the above files before running the below commands._
+Alternatively, you can also download the `schematic_service_account_creds.json` file, which uses the service account mode of authentication (_for Google services_). To download this file, run the following:
 
-For `credentials.json` file:
 ```bash
-synapse get syn21088684
+$ synapse get syn24214983
 ```
 
-For `quickstart-1560359685924-198a7114b6b5.json` file:
+Note: The service account mode of authentication is still being tested and certain functionalities (for our app) which leverage Google Apps Script API are [not fully compatible with this mode](https://developers.google.com/apps-script/api/concepts). So we recommend you follow the first command in this section.
```

Comment:
I would be more specific here. Isn't it just the `list` validation rule that doesn't work?